src/parquet/column/properties.h

```diff
@@ -81,8 +82,11 @@ static int64_t DEFAULT_PAGE_SIZE = 1024 * 1024;
 static int64_t DEFAULT_DICTIONARY_PAGE_SIZE = DEFAULT_PAGE_SIZE;
 static bool DEFAULT_IS_DICTIONARY_ENABLED = true;
 static Encoding::type DEFAULT_ENCODING = Encoding::PLAIN;
+static Encoding::type DEFAULT_DICTIONARY_ENCODING = Encoding::PLAIN_DICTIONARY;
```

Comment:
See https://github.com/apache/parquet-format/blob/master/Encodings.md
https://github.com/apache/parquet-format/blob/master/src/main/thrift/parquet.thrift#L285

I want to avoid mixing up the Parquet V1 and V2 dictionary encoding styles (I need to review Uwe's patch next), but I want to confirm my interpretation is correct 

In Parquet V1, the intent was to have

- Dictionary encoded data: value encoding type `PLAIN_DICTIONARY`. The assumption was that the dictionary was always PLAIN encoded and that the dictionary indices are RLE encoded. The encoding for the dictionary page was generally PLAIN. see

https://github.com/cloudera/Impala/blob/cdh5.8.0-release/be/src/exec/hdfs-parquet-table-writer.cc#L774

In Parquet V2, this was cleaned up so that it was more clear that the data pages are RLE encoded referencing some dictionary, but the dictionary can have any of the valid data page encodings, i.e.:

- Dictionary-encoded data pages, encoding is `RLE_DICTIONARY`
- Dictionary pages: any value encoding (except dictionary-encoding, of course)

is that right?